python/cuml/test/test_linear_model.py

```diff
@@ -119,6 +119,44 @@ def test_linear_models(datatype, X_type, y_type,
                            1e-1, with_sign=True)
 
 
+@pytest.mark.parametrize('datatype', [np.float32, np.float64])
+@pytest.mark.parametrize('X_type', ['ndarray'])
+def test_linear_models_default(datatype, X_type):
+    train_rows = np.int32(20*0.8)
+    X, y = make_regression(n_samples=(20), n_features=3,
```

Comment:
Why don't we add a pytest fixture for `small_regression_data` that automatically does `make_regression(20, 3)` followed by `train_test_split @ 20%` and returns it as a tuple? Seems like that gets used all over the place.